### PR TITLE
Plotting: add alias `--backtest-filename` for `--export-filename`

### PR DESCRIPTION
**Pull request by TheJoeSchr**
## Summary
makes it easier to discover how to use this argument

This is just an initial approach. If it seems to be good, I can also port it over to plot-dataframe etc.

If there is a better way to do this, please let me know...

### CODE REVIEW

File: freqtrade/commands/cli_options.py [hunk 1]
```diff
@@ -182,11 +182,17 @@ def __init__(self, *args, **kwargs):
...
-        'Example: `--export-filename=user_data/backtest_results/backtest_today.json`',
-        metavar='PATH',
+        "--export-filename",
+        help="Save backtest results to the file with this filename. "
+        "Requires `--export` to be set as well. "
+        "Example: `--export-filename=user_data/backtest_results/backtest_today.json`",
+        metavar="PATH",
+    ),
+    "backtestfilename": Arg(
+        "--backtest-filename",
```
Comment (xmatthias) [inline freqtrade/commands/cli_options.py]: I'd clearly not use 2 different arguments / configuration keys for the same thing.
That's pretty dangerous, and is bound to fail assuming one comes from the configuration (then you got both variables available - and will wonder why the wrong file is loaded).

you can assign a 2nd argument to the same key eventually, and maybe change the help text to something like `Filename for backtesting results" - so it works for both loading and saving.

**Review by xmatthias — CHANGES_REQUESTED** [review]

File: freqtrade/commands/cli_options.py (hunk 1 above)
Comment (TheJoeSchr) [reply freqtrade/commands/cli_options.py]: you mean like this? I haven't tested commit 298815d yet, I wanted to see if I understood you right

Comment (xmatthias) [reply freqtrade/commands/cli_options.py]: Yes exactly

**Comment by TheJoeSchr** [comment]
I think this is ready now?

I did this:

- rebased on current `develop`
- fixed flake8 linter issue
- tested `--backtest-filename` w/ `plot-profit`
- tested `--export-filename` w/ `plot-dataframe`
- tested `--backtest-filename` w/ `plot-profit`
- tested `--export-filenames` w/ `plot-dataframe`

**Review by xmatthias — APPROVED** [review]
For sure 👍

**Pull request merged**